plugin/src/main/java/org/opensearch/ml/task/MLTaskManager.java

```diff
@@ -39,6 +40,7 @@
 @Log4j2
 public class MLTaskManager {
     private final Map<String, MLTaskCache> taskCaches;
+    private final Map<String, List<String>> taskWorkNodesCaches;
```

Comment:
can we have some comment here to differentiate between` taskCaches` & `taskWorkNodesCaches`?